graspologic/__init__.py

```diff
@@ -19,6 +15,6 @@
 import graspologic.subgraph
 import graspologic.utils
 
-warnings.filterwarnings("ignore", category=RuntimeWarning)
```

Comment:
I am not sure why this was in the init originally but I agree that it is best practice to move it out to utils.py